.github/workflows/end-to-end-test-portals.yml

```diff
@@ -0,0 +1,86 @@
+name: End to end testing for Portals
+
+on:
+  push:
+  pull_request:
+    branches: [main]
+env:
+  PW_ALL_BLOBS_DIR: all-blob-reports
+  PORTALS_SUBDIR: apps/portals
+
+jobs:
+  e2e-tests:
+    if: ${{ github.ref_name == 'main' || github.actor == github.repository_owner }}
```

Comment:
Run e2e tests on push to `main` for the Sage repo and push to any branch for non-Sage repos